Car.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Car.hpp"
+#include <iostream>
+
+Car::Car() {
+    std::cout << __FUNCTION__ << " - ";
+}
+
+Car::~Car() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake(int speed) {
+    std::cout << __FUNCTION__ << ": ";
```

Comment:
How are you going to test the exception throwing of this function? If you try to use ASSERT_THROW(sth, invalidSth) unit test wont catch your exception.